docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -377,6 +414,31 @@ propagation from workload management workspace to mailbox workspace
 does not attempt to manage the resource and namespace definitions that
 are built into the mailbox workspaces.
 
+The above wording is deliberately restrained, for the sake of
+flexibility regarding resources that are defined one way in the edge
+clusters and another way in workload management workspace.  For
+example, the following scenario is allowed.
+
+- Some central team owns an API group and produces some
+  CustomResourceDefinition (CRD) objects that populate that API group.
+- That team derives APIResourceSchemas from those CRDs and a
+  corresponding APIExport of their API group.
+- That team maintains a kcp workspace holding those APIResourceSchemas
+  and that APIExport.
+- Some workload management workspaces have APIBindings to that
+  APIExport, and EdgePlacement objects that (1) select those
+  APIBinding objects for downsync and (2) select objects of kinds
+  defined through those APIBindings for either downsync or upsync.
+- Those resources are built into the edge clusters by pre-deploying
```

Comment:
Probably only the resources for upsync need to be pre-deployed, right ?  Won't the ones we downsync be deployed by the downsync itself ?

Reply:
Downsync cannot propagate APIBindings because the edge cluster is not a kcp workspace.
Also, an APIExport, and thus and APIBinding, is for a whole API group; one cannot use that mechanism to import a subset of what was exported.

Reply:
I was just referring to the "by pre-deploying"  part. Why do we need to pre-deploy into KCP resources that were coming from KCP in the first place?

Reply:
The point of this particular scenario is for the mailbox workspaces to have APIBindings to workload APIExports, so that the workload APIExport view holds the workload objects as they sit in the mailbox workspaces.  Particularly for upsynced resources, since this is something that the C2P people expected.